src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -108,13 +116,14 @@ export default function (props: SurveyTaskListProps) {
 					<div className="empty-message">{language("all-tasks-complete")}</div>
 				}
 				{tasks?.slice(0, props.limit).map((task) =>
-					variant == "multiCard" ? <Card key={task.id as string}>{getSurveyTaskElement(task)}</Card> : getSurveyTaskElement(task)
+					variant == "multiCard" ? <Card style={props.cardStyle} backgroundColor={resolveColor(context.colorScheme, props.cardBackgoundColor)} key={task.id as string}>{getSurveyTaskElement(task)}</Card> : getSurveyTaskElement(task)
 				)}
 			</div>
 		</TaskListWrapper>
 	);
 }
 
-function TaskListWrapper(props: { children?: React.ReactNode, card: boolean, innerRef?: React.Ref<HTMLDivElement> }) {
-	return props.card ? <Card innerRef={props.innerRef}>{props.children}</Card> : <div ref={props.innerRef}>{props.children}</div>;
+function TaskListWrapper(props: { children?: React.ReactNode, card: boolean, innerRef?: React.Ref<HTMLDivElement>, cardStyle?: React.CSSProperties, cardBackroundColor?: ColorDefinition }) {
```

Comment:
misspell cardBackroundColor

Reply:
ha apparently I misspelled twice - also called it "cardBackgoundColor" elsewhere